### PR TITLE
Update doc according to user feedbck

### LINKED ISSUE #31: Make user installation more fluent
Update doc to give user more hints, best to let user copy-and-paste the command in the doc to setup the playground.

### PR DESCRIPTION
**Pull request by lhw362950217**
Fix #31

### CODE REVIEW

File: dev.md [hunk 1]
```diff
@@ -11,11 +11,14 @@
     ```
 1. Run the `play.sh` under playgound's root directory.  This script will guide you to install SQLFlow on a virtualbox VM.  If you have a slow Internet connection to Vagrant Cloud, you might want to download the Ubuntu VirtualBox image manually from some mirror sites into `~/.cache/sqlflow/` before running the above script.  We use `get -c` here for continuing get the file from last breakpoint, so if this command fail, just re-run it.
```
Comment (llxxxll) [inline dev.md]: `get -c` > `wget -c`

File: dev.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ### For Developers
 
-1. Install VirtualBox and Vagrant on a computer with a relatively large memory size.
+1. Install [VirtualBox](https://www.virtualbox.org/) and [Vagrant](https://www.vagrantup.com/) on a computer with a relatively large memory size.
```
Comment (Yancey0623) [inline dev.md]: Can we list the minimal memory and CPU cores ?

File: dev.md (hunk 1 above)
Comment (lhw362950217) [reply dev.md]: done

File: dev.md (hunk 2 above)
Comment (lhw362950217) [reply dev.md]: I added a recommendation config

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**